ca/ca.go

```diff
@@ -162,7 +162,7 @@ func (a *Ca) PktTox509(pktCom []byte, caBytes []byte) ([]byte, error) {
 		return nil, err
 	}
 
-	err := client.VerifyPKToken(context.Background(), pkt, a.provider)
+	err := pkt.Verify(context.Background(), a.provider.CommitmentClaim())
```

Comment:
Why pass the CommitmentClaim()? Could we put the commitment claim in the pheader of the CIC signature?

Reply:
The more I think about your approach here the more I like it. It is nice way to capture the fact that location of the commitment is known by the provider.

Reply:
That was also my thinking but I have been waffling between it being internally provided. I don't like having this extra thing that people need to understand and implement but...

Reply:
> I don't like having this extra thing that people need to understand and implement

I believe you can take this approach and still abstract these details away from most people using OpenPubkey. The verifier can be supplied with OP configuration that includes the commitment claim, then most developers will never have be exposed to this.

```golang
v := verifierpkg.Verifier(verifierpkg.GoogleOP)

err:= v.VerifyPKToken(pkt)
```

Under the hood this will be calling `pkt.Verify(context.Background(), a.provider.CommitmentClaim())` but the developer doesn't need to see those details unless they are really mucking about and writing their own OP configs.